selinux/ipa.te

```diff
@@ -108,6 +108,19 @@ optional_policy(`
 	sssd_stream_connect(ipa_otpd_t)
 ')
 
+########################################
+#
+# oddjob local policy to enable ipa-helper
+#
+require {
+	type httpd_t;
+	type oddjob_t;
+	class process { noatsecure rlimitinh siginh transition };
+}
+
+allow oddjob_t httpd_t:process transition;
+allow oddjob_t ipa_helper_t:process { noatsecure rlimitinh siginh };
+
```

Comment:
A SELinux domain should not use external types directly like that. FreeIPA has to use interfaces and wrap them in an optional policy block.

Reply:
We are discussing what should be in selinux-policy and what should be in ipa right now with SELinux maintainers.